src/stringnumber.js

```diff
@@ -27,7 +27,11 @@ export class StringNumber extends Modifier {
     let shift_left = 0;
     let shift_right = 0;
 
-    let i, num, note, pos, props_tmp;
+    let i;
+    let num;
+    let note;
+    let pos;
+    let props_tmp;

```

Comment:
I agree with @aaronmars. Putting each declaration on a new line is definitely uglier but highlights a code smell.